content-src/components/Spotlight/Spotlight.scss

```diff
@@ -0,0 +1,56 @@
+.spotlight {
```

Comment:
The overflow-y: auto on .new-tab-wrapper is cutting off the bottom of the shadows on these, can you just remove that rule from `NewTabPage/NewTabPage.scss`? We don't want it to scroll anyway i think